requirements.txt

```diff
@@ -15,3 +15,5 @@ requests
 tenacity>=8.2.3
 tqdm
 ujson
+anyio==4.6.2.post1
```

Comment:
we can remove the version requirement, our code should work with any latest version

Reply:
Done!